src/main/java/org/jenkinsci/plugins/envinject/service/EnvInjectPasswordsMasker.java

```diff
@@ -18,19 +25,34 @@
 import javax.annotation.Nonnull;
 
 /**
+ * Masks {@link PasswordParameterValue}s
  * @author Gregory Boissinot
  */
 public class EnvInjectPasswordsMasker implements Serializable {
 
-
-    public void maskPasswordsIfAny(@Nonnull AbstractBuild build, @Nonnull EnvInjectLogger logger, @Nonnull Map<String, String> envVars) {
-        maskPasswordsJobParameterIfAny(build, logger, envVars);
-        maskPasswordsEnvInjectIfAny(build, logger, envVars);
+    /**
+     * @deprecated Use {@link #maskPasswordsIfAny(hudson.model.Run, java.util.Map, org.jenkinsci.lib.envinject.EnvInjectLogger)} 
+     */
+    @Deprecated
+    public void maskPasswordsIfAny(@Nonnull AbstractBuild run, @Nonnull EnvInjectLogger logger, @Nonnull Map<String, String> envVars) {
+        maskPasswordParameterssIfAny(run, envVars, logger);
+    }
+    
+    /**
+     * Masks {@link PasswordParameterValue}s.
+     * @param run Run
+     * @param envVars Target collection with Environment variables to be masked
+     * @param logger Logger
+     * @since 2.1
+     */
+    public void maskPasswordParameterssIfAny(@Nonnull Run<?, ?> run, @Nonnull Map<String, String> envVars, @Nonnull EnvInjectLogger logger) {
```

Comment:
http://geek-and-poke.com/geekandpoke/2015/1/12/what-every-good-system-needs . Migrated from EnvInject Lib, will fix